mod_ci/controllers.py

```diff
@@ -444,52 +445,39 @@ def kvm_processor(db, kvm_name, platform, repository, delay):
 
 def queue_test(db, gh_commit, commit, test_type, branch="master", pr_nr=0):
     """
-    Function to store test details into Test model for each platform, and post the status to GitHub.
-
-    :param db: Database connection.
-    :type db: sqlalchemy.orm.scoped_session
-    :param gh_commit: The GitHub API call for the commit. Can be None
-    :type gh_commit: Any
-    :param commit: The commit hash.
-    :type commit: str
-    :param test_type: The type of test
-    :type test_type: TestType
-    :param branch: Branch name
-    :type branch: str
-    :param pr_nr: Pull Request number, if applicable.
-    :type pr_nr: int
-    :return: Nothing
-    :rtype: None
+    Function to store test details into Test model separately for various
+    vm. Post status to GitHub
     """
     from run import log
-
-    fork = Fork.query.filter(Fork.github.like("%/CCExtractor/ccextractor.git")).first()
-
+    fork = Fork.query.filter(Fork.github.like(
+        "%/CCExtractor/ccextractor.git")).first()
     if test_type == TestType.pull_request:
         branch = "pull_request"
-
-    linux_test = Test(TestPlatform.linux, test_type, fork.id, branch, commit, pr_nr)
-    db.add(linux_test)
-    windows_test = Test(TestPlatform.windows, test_type, fork.id, branch, commit, pr_nr)
-    db.add(windows_test)
+    # Create Linux test entry
+    linux = Test(TestPlatform.linux, test_type, fork.id, branch, commit,
```

Comment:
You don't have to short each line. You can kept line length upto 90 characters. It will be easy to read.

Reply:
Are you sure? The PEP8 guidelines say 79 characters.

Reply:
Yes but on sample platform we have increased it to 90 characters, you can simply try pushing the changes upto 90 characters line.

Reply:
We already simplified all lines which are exceeding 79 characters in precious commit:
https://github.com/CCExtractor/sample-platform/commit/8403bae4583da989a96633ef6c05844ae6e5ecb1

Reply:
Will send another commit to that accord😃